packages/realm/src/scripts/submit-analytics.ts

```diff
@@ -122,17 +116,15 @@ function writePackageJson(packagePath, content) {
  * @returns true if analytics is disabled
  */
 function isAnalyticsDisabled() {
-  let isDisabled = false;
-
-  // NODE_ENV is commonly used by JavaScript framework
-  if ("NODE_ENV" in process.env) {
-    isDisabled |= process.env["NODE_ENV"] === "production" || process.env["NODE_ENV"] === "test";
+  if ("REALM_DISABLE_ANALYTICS" in process.env || "CI" in process.env) {
+    return true;
+  } else if (
+    // NODE_ENV is commonly used by JavaScript framework
+    process.env["NODE_ENV"] === "production" ||
+    process.env["NODE_ENV"] === "test"
+  ) {
+    return true;
   }
```

Comment:
I think you need a `return false` in the end - otherwise `isAnalyticsDsiabled()` will return `undefined`